swagger/index.css

```diff
@@ -1,16 +1,16 @@
 html {
-    box-sizing: border-box;
-    overflow: -moz-scrollbars-vertical;
-    overflow-y: scroll;
+  box-sizing: border-box;
+  overflow: -moz-scrollbars-vertical;
+  overflow-y: scroll;
 }
```

Comment:
note i wouldn't consider this change persistent (ie. still would need to ignore this folder) since this folder gets replaced from updated swagger packager every once in a while

Reply:
That's good to know, I'll include it in .stylelintingnore. Thanks!